src/VisualStudio/CSharp/Impl/EventHookup/EventHookupCommandHandler_ShadowedCommands.cs

```diff
@@ -12,11 +12,6 @@ internal partial class EventHookupCommandHandler : VSCommanding.ICommandHandler<
         public bool ExecuteCommand(InvokeCompletionListCommandArgs args, CommandExecutionContext context)
```

Comment:
Looks like this entire handler is dead now?

Reply:
deleted